### PR TITLE
README-Polish

### PR DESCRIPTION
**Pull request by mbiesiad**
Polish translation

### CODE REVIEW

**Comment by mbiesiad** [comment]
Hi there! Really great project. Here is translation into Polish, finished: https://github.com/mbiesiad/nodebestpractices
Best regards

**Comment by goldbergyoni** [comment]
@mbiesiad Wow, amazing work! 🏅 😲 

Just to ensure I get the status straight: By now, the 1st page is translated but not the inner pages? Do they refer to the English version?

**Comment by mbiesiad** [comment]
My pleasure! :-) yea, now I see. Yup, at this moment that's refered to the English version. Thank you so much for your note, please give me a time, I will let you know when it will be done. [WIP] Stay tuned!

**Comment by mbiesiad** [comment]
@goldbergyoni uff, done. Looks good for me, I hope there will not be any bigger fixes. Added & translated other subsites. I hope you enjoy! Greetings

File: .operations/writing-guidelines.polish.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Nasz manifest pisania treści
+
+Jak zwiększamy komfort czytania i uczenia się dla naszych gości.
+
+## 1. Proste jest lepsze niż lepsze
```
Comment (kekuu) [inline .operations/writing-guidelines.polish.md]: I think there should be simply: 'Proste jest lepsze'

File: .operations/writing-guidelines.polish.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Nasi czytelnicy powinni mieć wielką pewność, że przeglądane przez nich treści są wiarygodne. Osiągamy to poprzez włączenie dowodów, takich jak referencje, dane i inne zasoby dostępne na ten temat. Praktycznie staraj się zamieszczać cytaty z wiarygodnych źródeł, wykazywać wzorce, powiązane wzorce projektowe lub wszelkie środki naukowe, aby udowodnić swoje twierdzenia.
+
+## 3. MECE (Mutually Exclusive and Collectively Exhaustive)
+
+Oprócz tego, że treść jest znacznie edytowana i niezawodna, przeglądanie w niej powinno również zapewniać pełne omówienie tematu. Nie można pominąć żadnego ważnego tematu.
+
+## 4. Spójne formatowanie
+
+Treść jest prezentowana przy użyciu stałych szablonów. Wszelkie przyszłe treści muszą być zgodne z tym samym szablonem. Jeśli chcesz dodać nowe punktory, skopiuj format punktora z istniejącego i wypuść go do swoich potrzeb. Aby uzyskać dodatkowe informacje, zobacz [ten szablon](https://github.com/i0natan/nodebestpractices/blob/master/sections/template.md).
```
Comment (kekuu) [inline .operations/writing-guidelines.polish.md]: ' i wypuść go do ' should be " i rozszerz go do "

File: .operations/writing-guidelines.polish.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## 5. To na temat Node.js
+
+Każda rada powinna dotyczyć bezpośrednio Node.js, a nie ogólnie oprogramowania. Kiedy radzimy zaimplementować ogólny wzorzec / regułę w Node.js, treść powinna koncentrować się na implementacji Node. Na przykład, gdy ze względów bezpieczeństwa zalecamy sanitize danych wejściowych z żądań, należy użyć Node-lingo - „Użyj oprogramowania pośredniego do sanitize danych wejściowych”. Jeśli element nie ma określonej implementacji w Node.js (np. wygląda tak samo w Python i Jaba) - dołącz go do ogólnego elementu kontenera, patrz na przykład pozycja 6.5.
```
Comment (kekuu) [inline .operations/writing-guidelines.polish.md]: "sanitize danych wejściowych z żądań " change to: " ochronę danych wrażliwych z wejściowych żądań"

Comment (kekuu) [reply .operations/writing-guidelines.polish.md]: Same: „Użyj oprogramowania pośredniego do sanitize danych wejściowych” change to: „Użyj oprogramowania pośredniego do oczyszczenia danych wejściowych"

File: .operations/writing-guidelines.polish.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## 6. Tylko wiodący dostawcy
+
+Czasami przydatne jest podanie nazw dostawców, którzy mogą rozwiązać niektóre wyzwania i problemy, takie jak pakiety npm, narzędzia open source, a nawet produkty komercyjne. Aby uniknąć przytłaczających długich list lub rekomendować projekty niestabilne i niestabilne, opracowaliśmy następujące zasady:
```
Comment (kekuu) [inline .operations/writing-guidelines.polish.md]: "niestabilne i niestabilne" change to: "nierenomowane i niestabilne"

File: .operations/writing-guidelines.polish.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+- Polecamy tylko 3 najlepszych dostawców - dostawcę, który pojawia się w 3 najlepszych wynikach wyszukiwania (Google lub GitHub posortowane według popularności) dla danego odpowiedniego słowa kluczowego, możemy uwzględnić w naszej rekomendacji.
+- Jeśli jest to pakiet npm, należy go również pobrać średnio co najmniej 750 razy dziennie.
```
Comment (kekuu) [inline .operations/writing-guidelines.polish.md]: "Jeśli jest to pakiet npm, należy go również pobrać średnio co najmniej 750 razy dziennie." change to:  "musi być pobierany średnio co najmniej 750 razy dziennie."

File: sections/errorhandling/catchunhandledpromiserejection.polish.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
+# Złap nieobsłużone odrzucenia promises
+
+<br/><br/>
+
+### Wyjaśnienie jednego akapitu
+
+Zazwyczaj większość współczesnego kodu aplikacji Node.js / Express działa w ramach obietnic - czy to w .then handler, wywołaniu zwrotnym funkcji, czy w bloku catch. Zaskakujące jest to, że jeśli programista nie pamięta o dodaniu klauzuli .catch, błędy zgłaszane w tych miejscach nie są obsługiwane przez moduł obsługi zdarzeń uncaughtException i znikają. Najnowsze wersje Node dodały komunikat ostrzegawczy, gdy pojawia się nieobsługiwane odrzucenie, chociaż może to pomóc w zauważeniu, że coś pójdzie nie tak, ale oczywiście nie jest to właściwa metoda obsługi błędów. Prostym rozwiązaniem jest, aby nigdy nie zapomnieć o dodaniu klauzul .catch w każdym wywołaniu łańcucha obietnicy i przekierowaniu do scentralizowanej procedury obsługi błędów. Jednak budowanie strategii radzenia sobie z błędami wyłącznie w oparciu o dyscyplinę programisty jest dość kruche. W związku z tym zaleca się stosowanie płynnego wycofywania się i zapisanie się na `process.on ('unhandledRejection', callback)` - zapewni to, że każdy błąd promise, jeśli nie zostanie złapany lokalnie, zostanie obsłużony.
```
Comment (kekuu) [inline sections/errorhandling/catchunhandledpromiserejection.polish.md]: "że każdy błąd promise " change to: "że każdy błąd obietnicy" to be more consistent

File: sections/errorhandling/centralizedhandling.polish.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
...ów większe są szanse na poważne błędy ukryte pod radarem z powodu niewłaściwej obsługi. Obiekt obsługi błędów jest odpowiedzialny za uwidocznienie błędu, na przykład poprzez napisanie do dobrze sformatowanego programu rejestrującego, wysyłanie zdarzeń do jakiegoś produktu monitorującego, takiego jak [Sentry](https://sentry.io/), [Rollbar](https://rollbar.com/) lub [Raygun](https://raygun.com/). Większość frameworków internetowych, takich jak [Express](http://expressjs.com/en/guide/error-handling.html#writing-error-handlers), udostępnia mechanizm pośredniej obsługi błędów. Typowy przepływ obsługi błędów może być następujący: Niektóre moduły zgłaszają błąd -> router API łapie błąd -> propaguje błąd do oprogramowania pośredniego (np. Express, KOA), który jest odpowiedzialny za wychwytywanie błędów -> nazywa się scentralizowany moduł obsługi błędów -> oprogramowanie pośredniczące jest informowane, czy ten błąd jest niezaufanym błędem (nie działa), aby mógł z wdziękiem ponownie uruchomić aplikację. Pamiętaj, że częstą, ale niepoprawną praktyką jest obsługa błędów w oprogramowaniu pośrednim Express - takie postępowanie nie obejmie błędów zgłaszanych w interfejsach innych niż internetowe.
```
Comment (kekuu) [inline sections/errorhandling/centralizedhandling.polish.md]: "-> nazywa się scentralizowany moduł obsługi błędów ->" just: "-> scentralizowany moduł obsługi błędów ->"

File: sections/errorhandling/documentingusingswagger.polish.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
+# Dokumentuj błędy interfejsu API za pomocą Swagger lub GraphQL
+
+### Wyjaśnienie jednego akapitu
+
+REST API zwracają wyniki przy użyciu kodów stanu HTTP, użytkownik API musi bezwzględnie wiedzieć nie tylko o schemacie interfejsu API, ale także o potencjalnych błędach - osoba wywołująca może wtedy złapać błąd i taktownie go obsłużyć. Na przykład dokumentacja interfejsu API może z góry stwierdzać, że status HTTP 409 jest zwracany, gdy nazwa klienta już istnieje (zakładając, że interfejs API rejestruje nowych użytkowników), aby osoba wywołująca mogła odpowiednio wyświetlić najlepszy UX dla danej sytuacji. Swagger to standard definiujący schemat dokumentacji API oferujący ekosystem narzędzi umożliwiających łatwe tworzenie dokumentacji online, patrz ekrany drukowania poniżej
```
Comment (kekuu) [inline sections/errorhandling/documentingusingswagger.polish.md]: Forgotten dot at the end.

File: sections/errorhandling/operationalvsprogrammererror.polish.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
+# Rozróżnij błędy operacyjne i programistyczne
+
+### Wyjaśnienie jednego akapitu
+
+Rozróżnienie następujących dwóch typów błędów zminimalizuje przestoje aplikacji i pomoże uniknąć szalonych błędów: Błędy operacyjne odnoszą się do sytuacji, w których rozumiesz, co się stało i jego wpływ - na przykład zapytanie do jakiejś usługi HTTP nie powiodło się z powodu problemu z połączeniem. Z drugiej strony błędy programisty odnoszą się do przypadków, w których nie masz pojęcia, dlaczego, a czasem skąd pochodzi błąd - może to być jakiś kod, który próbował odczytać niezdefiniowaną wartość lub pula połączeń BD, która przecieka pamięć. Błędy operacyjne są stosunkowo łatwe w obsłudze - zwykle wystarczy zarejestrować błąd. Gdy pojawia się błąd programisty, rzeczy stają się włochate, aplikacja może być niespójna i nie ma nic lepszego niż zrestartować z wdziękiem
```
Comment (kekuu) [inline sections/errorhandling/operationalvsprogrammererror.polish.md]: "i jego wpływ" change to: "i ich wpływ"

Comment (kekuu) [reply sections/errorhandling/operationalvsprogrammererror.polish.md]: " która przecieka pamięć" should be: " z której wycieka pamięć"

Comment (kekuu) [reply sections/errorhandling/operationalvsprogrammererror.polish.md]: Last sentence: "Gdy pojawia się błąd programisty, rzeczy stają się zagmatwane, aplikacja może być niespójna i nie ma nic lepszego niż ją po prostu zrestartować"

File: sections/errorhandling/shuttingtheprocess.polish.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
+# Opuść ten proces z wdziękiem, gdy do miasta przybywa nieznajomy
+
+### Wyjaśnienie jednego akapitu
```
Comment (kekuu) [inline sections/errorhandling/shuttingtheprocess.polish.md]: "### Wyjaśnienie jednego akapitu" maybe change it to: "Wyjaśnienie w jednym akapicie"

File: sections/errorhandling/shuttingtheprocess.polish.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+
+Gdzieś w twoim kodzie obiekt procedury obsługi błędów jest odpowiedzialny za podjęcie decyzji, jak postępować, gdy błąd zostanie zgłoszony - jeśli błąd jest zaufany (tj. błąd operacyjny, zobacz dalsze wyjaśnienie w najlepszych praktykach #3), zapisywanie w pliku dziennika może być wystarczające. Sprawa staje się pokręcona, jeśli błąd nie jest znany - oznacza to, że niektóre komponenty mogą być wadliwe, a wszystkie przyszłe żądania mogą ulec awarii. Na przykład, zakładając singletonową, stateful token issuer service, która zgłosiła wyjątek i utraciła swój stan - od teraz może zachowywać się niespodziewanie i powodować niepowodzenie wszystkich żądań. W tym scenariuszu zabij proces i użyj „narzędzia Restarter” (takiego jak Forever, PM2 itp.), Aby zacząć od nowa z czystym stanem.
```
Comment (kekuu) [inline sections/errorhandling/shuttingtheprocess.polish.md]: "Na przykład, zakładając singletonową, stateful token issuer service, która zgłosiła wyjątek i utraciła swój stan" change to: "Na przykład, zakładając singletonowy serwis stateful token issuer, który zgłosił wyjątek i utracił swój stan"

File: sections/production/apmproducts.polish.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
+# Zapewnij użytkownikom wygodę korzystania z produktów APM
+
+<br/><br/>
+
+### Wyjaśnienie jednego akapitu
+
+APM (application performance monitoring) odnosi się do rodziny produktów, która ma na celu monitorowanie wydajności aplikacji od początku do końca, również z perspektywy klienta. Podczas gdy tradycyjne rozwiązania monitorujące koncentrują się na wyjątkach i niezależnych pomiarach technicznych (np. śledzeniu błędów, powolnych punktach końcowych serwera itp.), w prawdziwym świecie nasza aplikacja może tworzyć rozczarowanych użytkowników bez żadnych wyjątków kodu, na przykład, jeśli niektóre usługi oprogramowania pośredniego działają naprawdę wolno. Produkty APM mierzą wrażenia użytkownika od końca do końca, na przykład, biorąc pod uwagę system, który obejmuje interfejs użytkownika frontend i wiele usług rozproszonych - niektóre produkty APM mogą powiedzieć, jak szybko trwa transakcja obejmująca wiele poziomów. Może stwierdzić, czy wrażenia użytkownika są solidne i wskazać problem. Ta atrakcyjna oferta ma stosunkowo wysoką cenę, dlatego jest zalecana do dużych i złożonych produktów, które wymagają wykraczania poza proste monitorowanie.
```
Comment (kekuu) [inline sections/production/apmproducts.polish.md]: "od końca do końca," should be original: "end-to-end" or "na całej drodze"

File: sections/production/bestateless.polish.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
+# Bądź stateless, zabijaj Serwery prawie codziennie
```
Comment (kekuu) [inline sections/production/bestateless.polish.md]: "Serwery " change to: "serwery "

File: sections/production/frontendout.polish.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
+# Wyciągnij zasoby frontendu z Node'a
+
+<br/><br/>
+
+### Wyjaśnienie jednego akapitu
+
+W klasycznej aplikacji internetowej backend obsługuje frontend / grafikę w przeglądarce, bardzo powszechnym podejściem w świecie Node jest użycie Express'owego oprogramowania pośredniego do usprawnienia plików statycznych w kliencie. ALE - Node nie jest typową aplikacją internetową, ponieważ wykorzystuje pojedynczy wątek, który nie jest zoptymalizowany do obsługi wielu plików jednocześnie. Zamiast tego rozważ użycie odwrotnego proxy (np. Nginx, HAProxy), magazynu w chmurze lub CDN (np. AWS S3, Azure Blob Storage itp.), który wykorzystuje wiele optymalizacji tego zadania i uzyskuje znacznie lepszą przepustowość. Na przykład specjalistyczne oprogramowanie pośrednie, takie jak nginx, zawiera bezpośrednie przechwytywanie między systemem plików a kartą sieciową i wykorzystuje podejście wielowątkowe, aby zminimalizować interwencję między wieloma żądaniami.
```
Comment (kekuu) [inline sections/production/frontendout.polish.md]: "odwrotnego proxy" should be: "reverse-proxy"

File: sections/production/delegatetoproxy.polish.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
+# Deleguj wszystko, co możliwe (np. zawartość statyczną, gzip) do odwrotnego proxy
```
Comment (kekuu) [inline sections/production/delegatetoproxy.polish.md]: "odwrotnego proxy" should be: "reverse-proxy"

File: sections/production/frontendout.polish.md [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+
+Twoje optymalne rozwiązanie może mieć jedną z następujących postaci:
+
+1. Korzystanie z odwrotnego proxy - twoje pliki statyczne będą znajdować się tuż obok aplikacji Node, tylko żądania do folderu plików statycznych będą obsługiwane przez proxy, które znajduje się przed aplikacją Node, taką jak nginx. Dzięki takiemu podejściu aplikacja Node jest odpowiedzialna za wdrażanie plików statycznych, ale nie za ich obsługę. Twój kolega z interfejsu użytkownika pokocha to podejście, ponieważ zapobiega ono prośbom o pochodzenie z tego interfejsu.
```
Comment (kekuu) [inline sections/production/frontendout.polish.md]: "odwrotnego proxy" should be: "reverse-proxy"

File: sections/production/guardprocess.polish.md [hunk 17]
```diff
@@ -0,0 +1,17 @@
...
...uchamiania oraz bogatą integrację z węzłem. Inni z dużymi umiejętnościami w Linuksie mogą używać systemd i uruchamiać Node jako usługę. Sprawy stają się bardziej interesujące dla aplikacji korzystających z Dockera lub dowolnej technologii kontenerowej, ponieważ zwykle towarzyszą im narzędzia do zarządzania klastrami i organizacji (np. [AWS ECS](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html), [Kubernetes](https://kubernetes.io/) itp.), które wdrażają, monitorują i leczą pojemniki. Mając te wszystkie bogate funkcje zarządzania klastrami, w tym restart kontenera, po co mieszać się z innymi narzędziami, takimi jak PM2? Nie ma kuloodpornej odpowiedzi. Istnieją dobre powody, aby trzymać PM2 w kontenerach (głównie jego specyficzna wersja kontenerów [pm2-docker](https://www.npmjs.com/package/pm2-docker)) jako pierwsza warstwa ochronna - znacznie szybciej jest zrestartować przetwarzać i udostępniać funkcje specyficzne dla węzłów, takie jak oznaczanie kodu, gdy kontener hostujący prosi o wdzięczne ponowne uruchomienie. Inni mogą unikać niepotrzebnych warstw. Na zakończenie tego podsumowania żadne rozwiązanie nie pasuje do nich wszystkich, a poznanie opcji jest ważne
```
Comment (kekuu) [inline sections/production/guardprocess.polish.md]: "pojemniki" -> "kontenery"
Missing dot at the end.

File: sections/production/lockdependencies.polish.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
+# Zablokuj zależności
+
+<br/><br/>
+
+### Wyjaśnienie jednego akapitu
+
+Twój kod zależy od wielu zewnętrznych pakietów, powiedzmy, że „wymaga” i użyj momentjs-2.1.4, a następnie domyślnie po wdrożeniu do produkcji npm może pobrać momentjs 2.1.5, co niestety wprowadza kilka nowych błędów do tabeli. Użycie plików konfiguracyjnych npm i argumentu ``–save-exact = true`` instruuje npm, aby odwoływał się do *dokładnie* tej samej wersji, która została zainstalowana, więc następnym razem uruchomisz ``npm install`` (w wersji produkcyjnej lub w kontenerze Docker, który planujesz wysłać do testowania), zostanie pobrana ta sama zależna wersja. Alternatywnym i popularnym podejściem jest użycie pliku `.shrinkwrap` (łatwego do wygenerowania przy użyciu npm), który dokładnie określa, które pakiety i wersje powinny zostać zainstalowane, aby żadne środowisko nie mogło ulec pokusie pobierania nowszych wersji niż oczekiwano.
```
Comment (kekuu) [inline sections/production/lockdependencies.polish.md]: "i użyj momentjs-2.1.4" should be: "i używa momentjs-2.1.4,"
"więc następnym razem uruchomisz" should be: "więc gdy następnym razem uruchomisz"

File: .operations/writing-guidelines.polish.md (hunk 1 above)
Comment (mbiesiad) [reply .operations/writing-guidelines.polish.md]: agree, fixed. done, thanks a lot!

File: .operations/writing-guidelines.polish.md (hunk 2 above)
Comment (mbiesiad) [reply .operations/writing-guidelines.polish.md]: Agree, fixed. Done, thanks a lot!

File: .operations/writing-guidelines.polish.md (hunk 3 above)
Comment (mbiesiad) [reply .operations/writing-guidelines.polish.md]: Agree, fixed. Done, thanks a lot!

File: .operations/writing-guidelines.polish.md (hunk 4 above)
Comment (mbiesiad) [reply .operations/writing-guidelines.polish.md]: Agree, fixed. Done, thanks a lot!

File: .operations/writing-guidelines.polish.md (hunk 5 above)
Comment (mbiesiad) [reply .operations/writing-guidelines.polish.md]: Agree, fixed. Done, thanks a lot!

File: sections/errorhandling/catchunhandledpromiserejection.polish.md (hunk 6 above)
Comment (mbiesiad) [reply sections/errorhandling/catchunhandledpromiserejection.polish.md]: Agree, fixed. Done, thanks a lot!

File: sections/errorhandling/centralizedhandling.polish.md (hunk 7 above)
Comment (mbiesiad) [reply sections/errorhandling/centralizedhandling.polish.md]: Agree, fixed. Done, thanks a lot!

File: sections/errorhandling/operationalvsprogrammererror.polish.md (hunk 9 above)
Comment (mbiesiad) [reply sections/errorhandling/operationalvsprogrammererror.polish.md]: Agree, fixed. Done, thanks a lot!

File: sections/errorhandling/shuttingtheprocess.polish.md (hunk 10 above)
Comment (mbiesiad) [reply sections/errorhandling/shuttingtheprocess.polish.md]: Agree, fixed. Done, thanks a lot!

File: sections/errorhandling/shuttingtheprocess.polish.md (hunk 11 above)
Comment (mbiesiad) [reply sections/errorhandling/shuttingtheprocess.polish.md]: Agree, fixed. Done, thanks a lot!

File: sections/production/lockdependencies.polish.md (hunk 18 above)
Comment (mbiesiad) [reply sections/production/lockdependencies.polish.md]: Agree, fixed. Done, thanks a lot!

**Comment by mbiesiad** [comment]
@goldbergyoni updated, done. After CR. Enjoy! :-)

**Comment by goldbergyoni** [comment]
@mbiesiad  This is amazing work! ❤️ ❤️ ❤️ 

We will include this is the news section with your name & Tweet about it. Well, that's all we can do but maybe you can also share in local groups?

What's your Twitter handle?

We will soon also create a specific section for translators credit, for now adding you to the global credits list, I assume you also put your name in the translations section?

@all-contributors please add @mbiesiad  for content

**Comment by goldbergyoni** [comment]
@mbiesiad Oh, as I clicked 'Merge' I faced some conflict, can you help in resolving this?

**Comment by mbiesiad** [comment]
> 
> 
> @mbiesiad Oh, as I clicked 'Merge' I faced some conflict, can you help in resolving this?

Conflict info:
merge-conflict

Sure, I have added Polish flag etc to the README.md previously, sorry my fault. Now, I deleted this now to avoid merge conflict. Now should be OK, please let me know, if is.
That's a snippet for paste if you want to add later:
`Read in a different language: [CN**CN**](/README.chinese.md), [BR**BR**](/README.brazilian-portuguese.md), [RU**RU**](/README.russian.md), [PL**PL**](/README.polish.md), [(ES**ES**, FR**FR**, HE**HE**, KR**KR** and TR**TR** in progress!)](#translations)`

**Comment by mbiesiad** [comment]
Sounds great! Thank you! Sure, good idea, I will.
My Twitter handle: [@michalbiesiada](https://twitter.com/michalbiesiada)
Ok, sounds really nice! Do you mean this section? ("readme -> Translations - Completed translations").

**Comment by goldbergyoni** [comment]
@mbiesiad

Merged! 🍾🥂🍾🥂

Add yourself to the top of the Polish home page, and also to the path you referred to (finished translations)

I'm adding you to contributors and Tweeting about this. Cool?

**Comment by goldbergyoni** [comment]
@all-contributors please add @mbiesiad for content

**Comment by mbiesiad** [comment]
> 
> 
> @mbiesiad
> 
> Merged! 🍾🥂🍾🥂
> 
> Add yourself to the top of the Polish home page, and also to the path you referred to (finished translations)
> 
> I'm adding you to contributors, putting in news and Tweeting about this. Cool?

@goldbergyoni Sure, thank you! I really appreciate it. 🔥  Ok, sounds great! 🥇

**Pull request merged**